starter-code/clue.js

```diff
@@ -88,17 +88,67 @@ Patio
 
 */
 
-function random_selector(array) {
+var characters = [
+    {name:"Mrgreen", color:"Green"},
+    {name:"Mrorchid", color:"white"},
+    {name:"MrVictorPlum", color:"Purple"},
+    {name:"MissScarlet", color:"Red"},
+    {name:"MissPeacock", color:"blue"},
+    {name:"Mrmustard", color:"Yellow"},
+
+];
```

Comment:
Much more properties were expected for the characters. It's a good thing we don't need them.